tox.ini

```diff
@@ -4,7 +4,7 @@
 
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py27,py33,py34,py35,py36,py37,py38,py39,py310,pypy,flake8
+envlist = py27,py33,py34,py35,py36,py37,py38,py39,py310,py311,py312,pypy,flake8
```

Comment:
Adding newer python version is good, but we really need to get the test suite working first. Currently it fails with py2.7 and py < 3.8, probably because the versions are not availlable now in github actions.

It will be better to start with PR with this commit, and remove the missing python versions, or send another trivial PR wit these changes.

Reply:
Ok, let's try this: https://github.com/nirs/pytest-threadleak/pull/13